src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Listeners/ServiceBusScaleMonitor.cs

```diff
@@ -0,0 +1,337 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.ServiceBus;

```

Comment:
nit: sort/order these usings. Perhaps your VS settings for this aren't configured correctly (system namespaces first)?